README.md

```diff
@@ -13,21 +13,29 @@ and can be used with any editor that supports the protocol, to offer good suppor
 
 Features
 --------------
-* As you type reporting of parsing and compilation errors
+* Supports compiling projects from Java 1.5 through 15
+* Maven pom.xml project support
+* Limited Gradle support (Android projects are not supported)
+* Standalone Java files support
+* As you type reporting of syntax and compilation errors
 * Code completion
 * Javadoc hovers
-* Code actions / refactoring
+* Organize imports
+* Type search
+* Code/Source actions / Refactoring
```

Comment:
Looks strange only putting spaces around the latter `/`.

How about Code actions (quick fix, source actions & refactorings)

Reply:
Nice.